Firestore/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Unreleased
 
+# 1.3.1
+- [fixed] Disabling garbage collection now avoids even scheduling the
+  collection process. This can be used to prevent crashes in the background when
+  using `NSFileProtectionComplete`. Note that Firestore does not support
+  operating in this mode--nearly all API calls will fail while file protection
+  is enabled. This change just prevents a crash when Firestore is idle (#2846).
```

Comment:
What does "nearly all API calls will fail" mean?  I assume they'll trigger a crash rather than fail gracefully, right?  Should we be more explicit?

Reply:
I struggled with the language here quite a bit. I've changed "fail" to "cause crashes" does that help?